.ci/install.sh

```diff
@@ -12,6 +12,31 @@ else
   exit 1
 fi
 
+INSTALL_DIR="$HOME/.fnm"
+
+# Parse Flags
+parse_args() {
+  while [[ $# -gt 0 ]]
+  do
+  key="$1"
+
+  case $key in
+	  -d|--install-dir)
+	  INSTALL_DIR="$2"
+	  shift # past argument
+	  shift # past value
+	  ;;
+	  -s|--skip-shell)
+	  SKIP_SHELL="true"
+	  shift # past argument
+	  ;;
+  esac
```

Comment:
oh oh I just figured out -- we might need to have the following case:

```bash
    *)
    echo "Wrong argument provided: $key"
    exit 1
    ;;
```

so we won't have infinite loops on providing the wrong argument

Reply:
Oh dang. Good catch! No keep the comments coming. This is your project and I want to make sure I keep inline with your vision. I will go ahead and add some docs to the readme and add the breakout for unrecognized arguments.